commons/addons/clever-cloud-addons.md

```diff
@@ -1,299 +1,92 @@
 
-<table class="table table-bordered table-striped dataTable"><caption>MySQL pricing plans</caption>
-<tr>
-<th>Name</th>
-<th>Cache (Memory)</th>
-<th>Disk</th>
-<th>Conn. limit</th>
-<th>Price /mo</th>
-</tr> 
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">DEV</span></td>
-<td>SHARED</td>
-<td>10 MB</td>
-<td>5</td>
-<td>Free</td>
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">S</span></td>
-<td>SHARED</td>
-<td>256 MB</td>
-<td>10</td>
-<td>15 €</td>
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">M</span></td>
-<td>1 GB</td>
-<td>10 GB</td>
-<td>75</td>
-<td>45 €</td>
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">L</span></td>
-<td>2 GB</td>
-<td>200 GB</td>
-<td>500</td>
-<td>300 €</td>
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">XL</span></td>
-<td>8 GB</td>
-<td>500 GB</td>
-<td>750</td>
-<td>1600 €</td>
-</tr>
-<tr>
-<td class="cc-col__price "><span class="label cc-label__price label-info">XXL</span></td>
-<td>16 GB</td>
-<td>900 GB</td>
-<td>2000</td>
-<td>2200 €</td>
-</tr>
-</table>
+Clever Cloud provides multiple add-on to work with you application :
 
-### Migrating from an old database
+* [MySQL](mysql.md)
+* [PostgreSQL](postgresql.md)
+* [MongoDB](mongodb.md)
+* [FS buckets](fs_buckets.md)
+* [Cellar](cellar.md)
+* [Redis](redis.md)
+* [Redsmin](redsmin.md)
 
-Some applications require a populated database to run properly.  
-If you want to import your **SQL** dump, you can use several methods:
+## Create an add-on for your application
 
-1. <a href="https://dbms-pma.clever-cloud.com/">our WebGUI (PHP My Admin)</a>
-2. command line tool for MySQL administration
-3. any MySQL client such as MySQL Workbench.
+In order to create an add-on for your application, go to the [Clever Cloud Console](https://console.clever-cloud.com/).
+When you are in the main page of the console, go to the organisation in which you wan to create the add-on,
+for example your [personal space](https://console.clever-cloud.com/users/me).
 
+When you are in the organisation, click on **Add an add-on**. This space let you create and configure the
+add-on to follow your needs.
 
-<br/><br/>If you need to import a very large dump, please send an email to <support@clever-cloud.com>.
+First, choose which *type* of add-on you want to create. See above to have a list of available add-ons and the
+corresponding pages for a description and information.
 
+Now, select the plan you need for you add-on. You can find details about the pricing, the capacity of the add-on, ...
+on this page or in the corresponding documentation page.
 
-## PostgreSQL plans
+The third step offer you to choose with which application you want to link you add-on. Linking an add-on with an
+application add [environment variables](/admin-console/environment-variables.md) to the application.
```

Comment:
Linking an addon to an application will provide configuration to the application through environment variables